spec/reek/examiner_spec.rb

```diff
@@ -108,5 +108,42 @@
         expect(examiner.smells).to eq([])
       end
     end
+
+    context 'with an incomprehensible source that causes Reek to crash' do
+      let(:subject) do
```

Comment:
Can you replace this with either `subject do ...`, or (preferred) `let(:examiner) do`?